.github/workflows/js.yml

```
@@ -25,6 +25,7 @@ jobs:
       with:
         node-version: '14'
     - run: yarn
+    - run: yarn run downloadCustomTypes
```

```suggestion
    - run: yarn downloadCustomTypes
```
I'm pretty sure that `run` is not needed for yarn.